README.md

```diff
@@ -18,65 +18,100 @@ Pull requests are welcome.
 
 ### Installation
 
-Library will be linked automatically. 
+Library will be linked automatically.
 
 If you have `react-native < 0.60`, check [Linking Guide](https://github.com/mieszko4/react-native-zoom-us/tree/master/docs/LINKING.md)
 
 #### iOS
+
 Make sure you have appropriate description in Info.plist:
-* `NSCameraUsageDescription`
-* `NSMicrophoneUsageDescription`
-* `NSPhotoLibraryUsageDescription`
+
+- `NSCameraUsageDescription`
+- `NSMicrophoneUsageDescription`
+- `NSPhotoLibraryUsageDescription`
 
 ## Usage
+
 ```typescript
-import ZoomUs from 'react-native-zoom-us';
+import ZoomUs from "react-native-zoom-us";
 
 // initialize minimal
 await ZoomUs.initialize({
-  clientKey: '...',
-  clientSecret: '...',
-})
+  clientKey: "...",
+  clientSecret: "...",
+});
 
 // initialize with extra config
-await ZoomUs.initialize({
-  clientKey: '...',
-  clientSecret: '...',
-  domain: 'zoom.us'
-}, {
-  disableShowVideoPreviewWhenJoinMeeting: true
-})
-
+await ZoomUs.initialize(
+  {
+    clientKey: "...",
+    clientSecret: "...",
+    domain: "zoom.us",
+  },
+  {
+    disableShowVideoPreviewWhenJoinMeeting: true,
+  }
+);
 
 // Start Meeting
 await ZoomUs.startMeeting({
-  userName: 'Johny',
-  meetingNumber: '12345678',
-  userId: 'our-identifier',
+  userName: "Johny",
+  meetingNumber: "12345678",
+  userId: "our-identifier",
   zoomAccessToken: zak,
   userType: 2, // optional
-})
-
+});
 
 // Join Meeting
 await ZoomUs.joinMeeting({
-  userName: 'Johny',
-  meetingNumber: '12345678',
-})
+  userName: "Johny",
+  meetingNumber: "12345678",
+});
 
 // Join Meeting with extra params
 await ZoomUs.joinMeeting({
-  userName: 'Johny',
-  meetingNumber: '12345678',
-  password: '1234',
-  participantID: 'our-unique-id',
+  userName: "Johny",
+  meetingNumber: "12345678",
+  password: "1234",
+  participantID: "our-unique-id",
   noAudio: true,
   noVideo: true,
-})
+});
+
+// Join Meeting with zoom meeting url
+await ZoomUs.joinMeetingWithWebUrl("<zoom_meeting_url>");
 ```
 
+## In Meeting Events
+
+```typescript
+import ZoomUs from "react-native-zoom-us";
+
+// Get My User Meeting Info
+await ZoomUs.getMyUserMeetingInfo(): Promise<RNZoomMyselfMeetingInfo>
```

Comment:
`USEFUL to use`

Could be generic though for any user `getUserInfoById` with nullable id